view/frontend/layout/customer_account.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="customer_account_navigation">
+            <block class="Magento\Customer\Block\Account\SortLinkInterface" name="customer-account-navigation-my-credit-cards-link">
+                <arguments>
+                    <argument name="path" xsi:type="string">omise/methods/cards</argument>
+                    <argument name="label" xsi:type="string" translate="true">Omise Payment Methods</argument>
```

Comment:
I think we shouldn't put 'Omise' here - I think it should be the same as whatever the title of the page is. The customer does not care or maybe even know what Omise is, and the store is highly unlikely to have more than one card payment handler active that could cause confusion

Reply:
it should be omise, it is not general place, other payment methods will create their own subpage here.

Reply:
I strongly disagree. Have you ever seen a store with more than one card payment provider active at once?

Reply:
I have never once seen a store that stores card details mention the name of the provider they use for payments

Reply:
@guzzilar Thoughts?

Reply:
Agreed with @jonrandy. `Omise Payment Method` doesn't give any hints to users that they can look up their saved card here. (imagine that you are a customer, decided to save a card with www.starbucks.co then, you want to delete your card later on, where or which menu will you click to see your cards?)